advocacy_docs/partner_docs/EsriArcGISProandArcGISEnterprise/04-ConfiguringEsriArcGISProandEsriArcGISEnterprise.mdx

```diff
@@ -0,0 +1,179 @@
+
+In the previous section you configured PostGIS with your EDB Postgres Advanced Server instance and created some sample tables, as well as added some data to those tables to ensure that PostGIS was configured correctly.  In doing so, you have primed your database to be ready to interact with Esri ArcGIS Pro as you create an Enterprise Geodatabase.
+
+## Download GIS Data
+
+There are many sources to download GIS data from, Esri provides sample data, which is available at: https://hub.arcgis.com/datasets.
+
+We are using the sample 'HongKong GDB' data file for demonstration, which can be found here: https://www.arcgis.com/home/item.html?id=96a56c14f72c4638961f4ee9a271e34d.
+
+## Enable Enterprise Geodatabase
+
+1. Download the License File from My Esri
+
+  - Go to myesri.com and go to `Licenses` under My Organization, then click `Start` to begin the licensing process.
+  - Under Product select `ArcGIS Enterprise`.
+  - Under Version select the `Version 11`.
+  - Under License Type select `ArcGIS Server` and then select `Server Licensing roles and extensions` then select `Next`. 
+  - From there continue selecting the ArcGIS Server license configurations and enter your role information and the Organization information on the following pages.
+  - When you get to Review License File, ensure all of the information you input is correct and choose to receive this file by Download.
+
+2. Generate a keycodes file to authorize an Enterprise Geodatabase
+
+  - On the machine, navigate to the following location: `C:\Program Files (x86)\Common Files\ArcGIS\bin`.
+  - Double-click `SoftwareAuthorization.exe` to run the authorization wizard.
+  - In the Software Authorization Wizard, under `Authorization Options`, select `I have received an authorization file and am now ready to finish the authorization process`.
+
+![EsriAuthorization](Images/EsriAuthorization.png)
+
+  - Click `Browse...` to select the .ecp authorization file obtained from My Esri. 
+  - Select the file used to authorize the organization's ArcGIS Server machine.
+  - For the Product to be Authorized, ensure ArcGIS Server is selected, and click `Next`.
+  - Refer to Steps 7 to 14 of `ArcGIS Pro: Authorize a Single Use License` (https://pro.arcgis.com/en/pro-app/latest/get-started/authorize-and-start-arcgis-pro-with-a-single-use-license.htm) to complete the software authorization process.
+
+**The following photos show the Authorization Screens to step through for the Keycodes file to be finalized**
+
+![EsriAuthorizationMethod](Images/EsriAuthorizationMethod.png)
+
+![EsriAuthorizationChoices](Images/EsriAuthorizationChoices.png)
+
+![EsriAuthorizationExtensions](Images/EsriAuthorizationExtensions.png)
+
+  - Navigate to the following location to access the newly created keycodes file and verify the keycodes file is present: `C:\Program Files(x86)\Esri\License10.x\sysgen`.
+  - Authorize the enterprise geodatabase with the newly created keycodes file.
```

Comment:
How?  I thought I was authorizing the database.
At this point it might be better to say
1. Generate a keycode file to authorize your Enterprise Geodatabase accourding to the Esri docs.
2. Verify you have the key code typical located in C:\Program Files(x86)\Esri\License10.x\sysgen
3. Use the keycode to authorize your geo database.   

Do we have steps for this authorization?  if not then say authorize geo database accourding to esri docs.